### DISCUSSION TITLE
Reevaluate the ability to pipe from stdin to buffer on MacOS

### DISCUSSION THREAD

**Discussion by openclosure** [helix-editor/helix#6091] [Q&A]
Per PR #996 piping from MacOS had a bug due to https://github.com/crossterm-rs/crossterm/issues/500 and was disabled, however, it looks like this issue was actually fixed recently by https://github.com/crossterm-rs/crossterm/pull/735, as is mentioned in the original issue. Given the fix, is it possible for someone to check and see if this fixes the piping issue for helix as well? Could be an easy win to just re-enable it.

**Comment by gabydd** [comment]
Fixed by https://github.com/helix-editor/helix/pull/5468

**Reply by the-mikedavis** [reply]
In particular see https://github.com/helix-editor/helix/pull/5468#issuecomment-1438157344. I have tried #5468 on macos and it works well but we should probably wait for https://github.com/crossterm-rs/crossterm/issues/755 to be resolved upstream before enabling that feature flag.
